### ISSUE TITLE
Micro caching issue not fully solved

### ISSUE THREAD

**Issue by hubertus2017** [engintron/engintron#1076]
This should be enough:
```
if ($SITE_URI ~* "/checkout/") {
    # rules go here
}
```

_Originally posted by @fevangelou in https://github.com/engintron/engintron/issues/1053#issuecomment-496843338_

**Comment by hubertus2017** [comment]
Hello,

the micro caching issue seems not to be solved somehow.

History:
We run a magento webshop without any issues since 2 years.
After installation of eningtron, we faced some issues with customers seeing other customers orders, and mixed up order contents.

It was aid this is a problem of microcaching.

Then we added the following custom rule:

if ($SITE_URI ~* "domain-xy.de/admin_francop|domain-xy.de/customer|admin|domain-xy.de/checkout|domain-xy.de/catalogsearch|cpsess|domain-xy.de/contact")
{ 
set $CACHE_BYPASS_FOR_DYNAMIC 1; # Disables micro-caching 
}

Then we seem to had no more issues (obiously?!) the last 4 weeks. But today, again, i was doing a test order, and i saw in cart checkout the content of another customer, who wrote me he could not complete his order (because he disabled a paypal popup with his popup blocker).

Is there anything wrong with the syntax above, or did i forget anything to be prevented from caching?

Hubertus
